### PR TITLE
Audit Elevate NPSP Connector Installer and Product for Power of Us Hub links

### PR DESCRIPTION
**Pull request by balsamhindi**
[W-11647682](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000014wjyyYAA/view) (WI specifies Elevate but have changed it to be across all NPSP-based repos)
Performed audit and modification of POU links based on  https://docs.google.com/spreadsheets/d/1yqj16qZWe_LndjqvamQXINs_-XQ0ggjBDBpR-6EIhgo/edit#gid=1076789067
# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

**Review by ErinWiedemer — APPROVED** [review]
Looks good to me, @balsamhindi!

File: src/translations/iw.translation [hunk 1]
```diff
@@ -61,7 +61,7 @@
         <name>Contact_Merge_Step4</name>
     </customLabels>
     <customLabels>
-        <label>&apos;ערכת ההצלחה למלכ&quot;רים&apos; תומכת בשלושה מודלים שונים של חשבון: מודל חשבון משק הבית, מודל חשבון &apos;אחד לאחד&apos; ומודל החשבון האישי (או Bucket). למידע נוסף על מודלים אלה, עיין &lt;a href=&quot;https://powerofus.force.com/NPSP_Account_Model&quot; target=&quot;_blank&quot;&gt;במסמכי התיעוד של NPSP‏&lt;/a&gt;.‏
+        <label>&apos;ערכת ההצלחה למלכ&quot;רים&apos; תומכת בשלושה מודלים שונים של חשבון: מודל חשבון משק הבית, מודל חשבון &apos;אחד לאחד&apos; ומודל החשבון האישי (או Bucket)
```
Comment (ErinWiedemer) [inline src/translations/iw.translation:64]:
```suggestion
        <label>&apos;ערכת ההצלחה למלכ&quot;רים&apos; תומכת בשלושה מודלים שונים של חשבון: מודל חשבון משק הבית, מודל חשבון &apos;אחד לאחד&apos; ומודל החשבון האישי (או Bucket).
```

**Review by ErinWiedemer — APPROVED** [review]
Hi @balsamhindi, all of this looks good, though I did add back a period on the Hebrew translation.

**Comment by balsamhindi** [comment]
> Hi @balsamhindi, all of this looks good, though I did add back a period on the Hebrew translation.

Thank you @ErinWiedemer ! I've accepted that change, yeah good **point** the Hebrew was interesting to modify because the text is meant to be viewed right to left, hope I got that modified properly!

**Review by daniel-fuller — APPROVED** [review]

**Pull request merged**